pymc/pytensorf.py

```diff
@@ -47,6 +46,7 @@
 from pytensor.graph.fg import FunctionGraph
 from pytensor.graph.op import Op
 from pytensor.scalar.basic import Cast
+from pytensor.scalar.basic import identity as scalar_identity
```

Comment:
You don't need to create a new Elemwise, there's already one defined in `tensor.math` (or basic), just called `tensor_copy`